app/api/healthcareAuthorities/getConfigurationApi.ts

```diff
@@ -0,0 +1,24 @@
+import type { HealthcareAuthority } from './getHealthcareAuthoritiesApi';
+
+export type Configuration = {
+  name: string;
+  infoWebsiteUrl: string;
+  privacyPolicyUrl: string;
+  referenceWebsiteUrl: string;
+  notificationThresholdPercent: number;
+  notificationThresholdCount: number;
```

Comment:
There was a recent [change in spec](https://pathcheck.atlassian.net/wiki/spaces/TEST/pages/81231882/Hashing+of+GPS+points+-+Design#Notification-of-sensitivity-controls), the `count` will soon be replaced by `timeframe` because the `count` doesn't say anything about the period of time in which those matches occurred.

Reply:
Nice catch -- I left a comment on the spec to make sure the configuration endpoint gets updated appropriately